assets/javascripts/discourse/components/tds-new-user.js.es6

```diff
@@ -1,17 +1,18 @@
+import { solidarity } from '../constants';
+
 export default Ember.Component.extend({
   // isDebtor and isInSolidarity control the visibility of the user fields
   isDebtor: false,
   isInSolidarity: false,
 
   actions: {
     collectiveSelected(event) {
-      const solidarityBloc = "Solidarity Bloc"
 
       const elements = $(".collective-selectors input:checked").toArray()
       const collectives = elements.map(element => element.attributes.value.value)
       
-      Ember.set(this, 'isInSolidarity', collectives.includes(solidarityBloc))
-      Ember.set(this, 'isDebtor', collectives.filter(x => x != solidarityBloc).length > 0)
+      Ember.set(this, 'isInSolidarity', collectives.includes(solidarity))
+      Ember.set(this, 'isDebtor', collectives.filter(x => x != solidarity).length > 0)
```

Comment:
You can use Array.prototype.find here instead.

Reply:
Cool, I didn't know that. I guess there is a small performance benefit (?) to using find, but I think I'll leave it as is because I think it's more readable this way.

Reply:
Or maybe it isn't. I can't decide.